doc/classes/CanvasItem.xml

```diff
@@ -302,7 +302,7 @@
 		<method name="get_local_mouse_position" qualifiers="const">
 			<return type="Vector2" />
 			<description>
-				Returns the mouse position relative to this item's position.
+				Returns the mouse's position reltative to the origin of this [CanvasItem].
```

Comment:
```suggestion
				Returns the mouse's position relative to the origin of this [CanvasItem].
```

Reply:
Changed to reflect changes agreed in the master branch version.